README.md

```diff
@@ -163,6 +163,11 @@ Ethereum nodes with exposed APIs! Further, all browser tabs can access locally
 running web servers, so malicious web pages could try to subvert locally available
 APIs!**
 
+### Tests
+```
+go test github.com/maticnetwork/bor/consensus/bor_test
```

Comment:
Instead of doing this, can we add [here](https://github.com/maticnetwork/bor/blob/master/Makefile#L32)

Reply:
Done. Thanks!